doc/source/getting_started/install.rst

```diff
@@ -360,6 +360,8 @@ zlib                                         Compression for HDF5
 fastparquet               0.4.0              Parquet reading / writing
 pyarrow                   0.15.0             Parquet, ORC, and feather reading / writing
 pyreadstat                                   SPSS files (.sav) reading
+pyreadr                                      R files (.RData, .rda, .rds) reading / writing
```

Comment:
@ParfaitG I didn't follow the entire thread, but we do not want to add these deps generally. IIRC you had a much simpler way (to just link in the c-code to read the format). that would be *much more prefereable*.

Reply:
> This PR relies on a new dependency, pyreadr, (available in conda) for default engine option. Please advise how to add to builds for pytests.

this would be ok. we do not want to add r as a dep for even testing.

Reply:
Due to maintenance and licensing, the consensus was to use `pyreadr` as a soft dep like `pyreadstat` for `read_spss`. Understood about not adding R as dep. For rscript engine, I am suggesting we use R via `subprocess` call similar to backends in `io.clipboard`. (Can serve as use case for Python/R arrow project). But this PR is set up to easily remove either engine (i.e., separate classes and tests). 

However, unless I am mistaken the CI tests does have R installed. I am getting results and fixing fails in `test_rscript.py` on Linux/Windows/Mac builds which checks for `Rscript` (nothing yet for `test_pyreadr.py`).

Reply:
> pyreadr as a soft dep like pyreadstat for read_spss.

yes this is ok

Reply:
>  For rscript engine, I am suggesting we use R via subprocess call similar to backends in io.clipboard. (Can serve as use case for Python/R arrow project). But this PR is set up to easily remove either engine (i.e., separate classes and tests).

I really don't want to complicate our CI any more. So i don't want this. Instead use simple frames for the expected return values).

Reply:
Got it. Will adjust code and docs to exclusively run pyreadr. Should I add `pyreadr` entries to the three yaml files in pandas/tree/master/ci/deps where `pyreadstat` is also included? Also, because .RData and .rda can potentially have more than one named data frame (unlike .rds), we may have to return a `dict` of DataFrames like `pyreadr` does.

Reply:
> Got it. Will adjust code and docs to exclusively run pyreadr. Should I add `pyreadr` entries to the three yaml files in pandas/tree/master/ci/deps where `pyreadstat` is also included? Also, because .RData and .rda can potentially have more than one named data frame (unlike .rds), we may have to return a `dict` of DataFrames like `pyreadr` does.

yes you want to add it by default so its used. but also one build should *not* have it so that if its not installed it skips properly.

> we may have to return a dict of DataFrames like pyreadr does.

that would be ok, we do this in `read_html`, just clearly document & make the signature reflect this.